01-syntax/03-data-structures/deep-comparation/deepEqual.js

```diff
@@ -4,15 +4,46 @@
  */
 /* global deepEqual */
 
-// → Your code here.
-
 var obj = {
   here: {
     is: 'an'
   },
   object: 2
 };
 
+function deepEqual(firstObj, secondObj) {
+  var theSame = false;
+  var firstKey;
+  var secondKey;
+
+  if (Object.keys(firstObj).length !== Object.keys(secondObj).length) {
```

Comment:
What if I would call 'deepEqual' for `obj` and `null` ?

Take care about that case